src/content/docs/style-guide/writing-docs/processes-procedures/create-edit-content.mdx

```diff
@@ -35,6 +35,13 @@ A member of the Docs team will review your pull request and comment with any fee
 
 If you'd like to go deeper with development, see our [Contributors guide](https://github.com/newrelic/docs-website/blob/develop/CONTRIBUTING.md#editing-existing-pages) for information on how to fork our site, build it locally, and submit pull requests.
 
+Here's a video that will walk you through the process step by step:
```

Comment:
@homelessbirds since this is such a short video, what do you think about inserting it before the bulleted list? For example:

To edit existing content without building the site locally, check out this short video or follow the steps below:

INSERT_VIDEO_HERE

Reply:
Great idea!